config/prow/config.yaml

```diff
@@ -130,12 +130,26 @@ branch-protection:
             users: []
             teams: []
           enforce_admins: true # protections apply to admins as well
+        gardener:
+          protect: true
+          required_status_checks:
+            contexts:
+            - license/cla
```

Comment:
Shouldn't we add the `concourse-ci/publish` step here as well to prevent PRs with failing image builds are merged?

Also, what about our GitHub actions?

Reply:
We could add `concourse-ci/publish`, yes.

I also thought about making the GitHub actions required. However, `Commit Message Check` is already done by prow's plugin `invalidcommitmsg`. And the `Milestone Check` will hopefully be replaced by tide merge blockers. For now, we could add it however.

Reply:
that's a good point, I forgot to mention that. `Commit Messsage Check` could be disabled, I did not completely understand what the `Milestone Check` is doing. That's why there is no tide merge blocker yet.

Reply:
I added 
```
- concourse-ci/publish
- check-release-milestone
```

Tide merge blocking issues seem to block all or nothing, so this won't allow to "cherry-pick" some PRs for a release. That's why the `check-release-milestone` check is still enforced. 
I can't really see if the name is right? I have chosen the job name from the github action. Could you verify that in github branch protection settings?

Reply:
hmmm, I cannot find event with a `*milestone*` context in the logs of hook
consourse events produce logs entries here 🤔

Reply:
This is what I got from the API:

```json
{
  "enabled": true,
  "required_status_checks": {
    "enforcement_level": "non_admins",
    "contexts": [
      "license/cla",
      "concourse-ci/publish",
      "Check Release Milestone",
      "Check Commit Messages",
      "pull-gardener-unit",
      "pull-gardener-integration"
    ],
    "checks": [
      {
        "context": "license/cla",
        "app_id": null
      },
      {
        "context": "concourse-ci/publish",
        "app_id": null
      },
      {
        "context": "Check Release Milestone",
        "app_id": null
      },
      {
        "context": "Check Commit Messages",
        "app_id": null
      },
      {
        "context": "pull-gardener-unit",
        "app_id": 154614
      },
      {
        "context": "pull-gardener-integration",
        "app_id": 154614
      }
    ]
  }
}
```

Reply:
ah, that's the name of the action 😅

Reply:
btw. I just noticed there is `"enforcement_level": "non_admins",` in your API result, but we set `enforce_admins: true # protections apply to admins as well`
Thus, prow would try to change the settings. Should we leave it like this?

Reply:
Yes, let's leave it like this 👍